opts/opts_test.go

```diff
@@ -445,7 +446,14 @@ func TestValidateHost(t *testing.T) {
 		"tcp://invalid":      "Invalid bind address format: invalid",
 		"tcp://invalid:port": "Invalid bind address format: invalid:port",
 	}
+	const defaultHTTPHost = "tcp://127.0.0.1:2375"
+	var defaultHOST = "unix:///var/run/docker.sock"
+
+	if runtime.GOOS == "windows" {
```

Comment:
I'm willing, but tbh, I think it would make it more likely that the test is updated on one platform, and not the other - the test is the same, its only the precondition that changes for windows.